File: Programmers/level2/멀쩡한 사각형.cpp
using namespace std;

long long solution(int w,int h) {
    long long answer = 0;
    double pos = 0;
    
    for(int i=1; i<w; i++)
    {
        pos = -((double)h / (double)w) * i + h;
        answer += (int)pos;
    }
    answer *= 2;
    return answer;
}

File: Programmers/Java/level2/멀쩡한 사각형.java
class Solution {
    public long solution(int w, int h) {
        long answer = 0;
        double cnt = 0;
        
        for(int i=1; i<w; i++)
        {
            cnt = -((double)h / (double)w) * i + h;
            answer += (int)cnt;
        }
        answer *= 2;
        return answer;
    }
}
